includes/admin.form.inc

```diff
@@ -76,6 +76,61 @@ function islandora_large_image_admin(array $form, array &$form_state) {
       ),
     ),
   );
+  $form['islandora_large_image_thumbnail_fieldset'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Thumbnail'),
+    '#description' => t('Settings for creating thumbnail derivatives<br/><span style="color:red">WARNING: This will set the size for all medium sized images generated as a derivative. Regenerate derivatives to apply this setting to existing objects.</span>'),
```

Comment:
Ooops, yep that needs to say TN. My thoughts were that thumbnails for collection and/or the site in general should probably be consistently generated  throughout the site and not very depending on the solution pack. I went with PDF's variable name so that this improvement wouldn't break or clash with an existing stack.